plugins/tiddlywiki/multiwikiserver/modules/startup.js

```diff
@@ -55,4 +56,8 @@ ServerManager.prototype.createServer = function(options) {
 	return server;
 }
 
+ServerManager.prototype.toggleServerAuth = function(shouldUseAuth = false){
```

Comment:
I think the name of the function is not right. IMO it should be `setServerAuth`.